fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -125,14 +115,7 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
                                             from = BrowserDirection.FromTranslationsDialogFragment,
                                         )
                                     },
-                                    onTranslateButtonClick = {
-                                        interactor.onTranslate(
-                                            tabId = args.sessionId,
-                                            fromLanguage = null,
-                                            toLanguage = null,
-                                            null,
-                                        )
-                                    },
+                                    onTranslateButtonClick = {},
```

Comment:
Did you intend on only removing the Interactor/Controller in this PR?

Reply:
That's right, I was also going to do [use case itself too](https://searchfox.org/mozilla-mobile/rev/99576d6e69eac8ee1bf6c8a654d5fa96a6f6bf84/firefox-android/android-components/components/feature/session/src/main/java/mozilla/components/feature/session/SessionUseCases.kt#469,499), but forgot reference browser was using the [use cases](https://searchfox.org/mozilla-mobile/rev/063c6bcf58858b26ebdc8cd0c8f2c658d82061da/firefox-android/android-components/samples/browser/src/main/java/org/mozilla/samples/browser/DefaultComponents.kt#326-338,344).

Reply:
(Mostly trying to [revert to this state](https://github.com/mozilla-mobile/firefox-android/pull/5148/files), sans getting rid of the id passing.)